oauth2_provider/views/application.py

```diff
@@ -37,6 +37,7 @@ def get_form_class(self):
                 "client_type",
                 "authorization_grant_type",
                 "redirect_uris",
+                "post_logout_redirect_uris",
```

Comment:
Is meant to be this a plural list of uris or singular? If the latter, please change the name to singular `post_logout_redirect_uri`

Reply:
Also the [test is failing](https://github.com/jazzband/django-oauth-toolkit/actions/runs/5246478141/jobs/9475365332?pr=1285#step:7:129)